modelci/ui.py

```diff
@@ -26,21 +27,21 @@ def single_model_view(model: Optional[dict], top=False):
         return ''
 
     if top:
-        model_name = f'[bold]{model["name"]}[/bold]'
+        model_name = f'[bold]{model["architecture"]}[/bold]'
     else:
-        model_name = model['name']
+        model_name = model['architecture']
 
     return (
```

Comment:
we may organize and format the parameters of the return. Not its is little chaos.

Better add a todo